docs/dask_cudf/Makefile

```diff
@@ -3,8 +3,9 @@
 
 # You can set these variables from the command line, and also
 # from the environment for the first two.
-SPHINXOPTS    ?=
+SPHINXOPTS    ?= -n -v
```

Comment:
Is there a reason why these flags don't match the `cudf` Makefile flags?

Will warnings be treated as errors for `dask-cudf` in a follow-up PR?

Reply:
Yes, I didn't want to also try and fix dask-cudf issues in the same PR. This one is large enough already. I also don't know dask-cudf as well so may want to get someone more familiar to work through those issues.